lib/osf-components/addon/components/file-actions-menu/template.hbs

```diff
@@ -48,5 +48,17 @@
                 {{t 'general.share'}}
             </dd.trigger>
         </SharingIcons::Dropdown>
+        <OsfLink
+            data-test-rename-button
+            data-analytics-name='Rename file'
+            aria-label={{t 'file_actions_menu.rename'}}
+            local-class='RenameLink'
+            value={{t 'file_actions_menu.rename'}}
+            @href='#userInput'
```

Comment:
I don't think you need `value` or `@href`

Reply:
Value is used to pre-populate the Input field from the original file name. It seems to work better than using placeholder since by persisting the filename the user is kept aware of it. A placeholder version is available on branch feature/rename-file-file-detail and feature/rename-file-file-list if you would like to see how it would differ. If you think this would be a better direction let me know but we would need one or the other. Similarly, @href can be replaced with @route which resolves the url redirect issue I was having while testing but you do need one or the other or the compiler turns to lava.

Reply:
Ok, this is now resolved.